README.md

```diff
@@ -8,18 +8,22 @@ The best part? It only takes a few lines of code:
 import json
 from data_profiler import Data, Profiler
 
-data = Data("your_file.csv") # CSV, AVRO, Parquet, JSON, DataFrame
+data = Data("your_file.csv") # Auto-Detect & Load: CSV, AVRO, Parquet, JSON, DataFrame
```

Comment:
Should we specify `auto detect file type`?

Reply:
I was trying to keep it short, IMO this may still be too long. Might be better with just "Detect & load"

I think it's fairly clear its file type, but 🤷‍♂️ anyone else have an opinion. I'm open to suggestions, just would like to keep it as short as possible.

Reply:
i think it is fine bc you reference the file types after you mention it, however, if @AnhTruong found that confusing, he may not be the only one. @AnhTruong did you not understand when you read it or are you just saying you think it needs more description?

Reply:
What about txt?

Reply:
`DataFrame` not detected